designer/client/src/components/graph/node-modal/editors/expression/AceWrapper.tsx

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable i18next/no-literal-string */
+import "ace-builds/src-noconflict/ace";
```

Comment:
1. Is it a specific vase to use this variant of this file? If i understand correctly it needs to resolve specific issue https://stackoverflow.com/a/77234551
2. Can we use the minified version of this file `ace-builds/src-min-noconflict/ace`

Reply:
webpack minimizes it anyway

Reply:
What about point 1 ? Do you know, if we have a specific issue that we need to use this version of ace-builds

> webpack minimizes it anyway

Yes you are right, but in the production build only, but generally, it doesn't matter, so we can just leave it.

Reply:
not changing anything except the order in the import tree. just same as others. now I moved it next to others in ace.ts